.github/workflows/update-commit-hashes.yml

```diff
@@ -26,12 +26,3 @@ jobs:
     secrets:
       MERGEBOT_TOKEN: ${{ secrets.MERGEBOT_TOKEN }}
       PYTORCHBOT_TOKEN: ${{ secrets.GH_PYTORCHBOT_TOKEN }}
-
-  update-vision-commit-hash:
```

Comment:
We should maybe just comment this out and leave a link to an issue describing the problem we were encountering so we can make a note to fix this later